core/templates/pages/volunteer-page/volunteer-page.component.ts

```diff
@@ -157,77 +174,152 @@ export class VolunteerPageComponent implements OnInit, OnDestroy {
+        '/volunteer/profile_images/diana.jpg'
       ],
       caption: [
         {
           content: (
-            '“Oppia has given me the opportunity to profoundly ' +
-            'impact the lives of many underprivileged children by ' +
-            'providing them with free quality education. ' +
-            'I am grateful to be a part of this venture.“'),
-          name: '- Riya Sogani',
-          type: 'Learner Feedback Program Volunteer, India'
+            'I contribute to Oppia because I believe in its cause. ' +
+            'Every child can do well at math if they’re taught in the right ' +
+            'way and it is broken down in a language they understand. Also, ' +
+            'no matter how good a product is, people will not know it ' +
+            'without effective marketing.'),
+          name: '- Yiga',
+          type: 'Marketing team'
+        },
+        {
+          content: (
+            '“Oppia\'s mission and values inspire me daily. I volunteer ' +
+            'because everyone deserves an education and helping to market ' +
+            'this opportunity is a privilege“'),
+          name: '- Jennifer',
+          type: 'Marketing team'
+        },
+        {
+          content: (
+            '“I love to tell stories through content creation. This not only' +
+            'amplifies Oppia\'s mission of providing free accessible ' +
+            'education but also fosters a sense of community and inspiration,' +
+            'encouraging more individuals to engage with Oppia\'s ' +
+            'educational resources.”'),
+          name: '- Erio',
+          type: 'Video creation team'
         },
         {
           content: (
-            '“Oppia has given me the chance to feel that I am ' +
-            'causing a positive ripple effect around me. ' +
-            'It is delightful to see underprivileged children\'s lives ' +
-            'improving because of Oppia\'s lessons. ' +
-            'These lessons taught them more than just mathematics in joyful ' +
-            'ways: they enhanced their confidence, technical skills, and ' +
-            'reading and writing skills!“'),
-          name: '- Wala Awad',
-          type: 'Learner Feedback Program Volunteer, Palestine'
+            '“I contribute to Oppia because seeing the community ' +
+            'and the impact it creates makes me hopeful for the future of ' +
+            'education, and I want to be a part of that change.“'),
+          name: '- Diana Chen',
+          type: 'Product Manager'
         }
       ]
     };
 
     this.lessonCreation = {
       images: [
+        '/volunteer/profile_images/aanuoluwapo.jpg',
+        '/volunteer/profile_images/viksar.jpg',
+        '/volunteer/profile_images/jackson.jpg',
+      ],
+      caption: [
+        {
+          content: (
+            '“At Oppia, my love for writing stories as a ' +
+            'creative intertwines with my passion for ' +
+            'education. “'),
```

Comment:
There is an extra space before the endquote, and I think the endquote is a different character than those used in other quotes?

Also, if the quotes are the same across all the blurbs then make that part of the HTML instead and omit them from the data.

Reply:
Done